packages/apiclient-services-subscribewithgoogle/SubscribewithGoogle.php

```diff
@@ -0,0 +1,461 @@
+<?php
+/*
+ * Copyright 2014 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+/**
+ * Service definition for SubscribewithGoogle (v1).
+ *
+ * <p>
+ * The Subscribe with Google Publication APIs enable a publisher to fetch
+ * information related to their SwG subscriptions, including the entitlement
+ * status of users who are requesting publisher content and the publications
+ * owned by the publisher.</p>
+ *
+ * <p>
+ * For more information about this service, see the API
+ * <a href="https://developers.google.com/news/subscribe/guides/overview" target="_blank">Documentation</a>
+ * </p>
+ *
+ * @author Google, Inc.
+ */
+class Google_Service_SubscribewithGoogle extends Google_Service
+{
+  /** See your primary Google Account email address. */
+  const USERINFO_EMAIL =
+      "https://www.googleapis.com/auth/userinfo.email";
```

Comment:
@aaemnnosttv This isn't used anywhere, is it? Just asking due to the lack of the actual SwG publications-related scope, which apparently is necessary.

If having this here actually matters, we may want to hard-code it? But I don't see currently why we would need to.

Reply:
No, it's a mistake somewhere I believe as _using_ the publications API does indeed require the SwG scope. I raised this with the API team and they're following up on that internally, but it shouldn't affect us as we don't rely on the discovery document to determine the scopes we need.